app/config/ConfigDecorator.scala

```diff
@@ -255,8 +248,8 @@ class ConfigDecorator @Inject()(
   def getFeedbackSurveyUrl(origin: Origin): String =
     feedbackSurveyFrontendHost + "/feedback/" + enc(origin.origin)
 
-  def getCompanyAuthFrontendSignOutUrl(continueUrl: String): String =
-    companyAuthHost + s"/gg/sign-out?continue=$continueUrl"
+  def getBasGatewayFrontendSignOutUrl(continueUrl: String): String =
+    basGatewayFrontendHost + s"/bas-gateway/sign-out-without-state?continue=$continueUrl"
```

Comment:
@vimalsuresh @anujakhare sign out should redirect them to this new bas-gateway url with /sign-out-without-state? in. I have checked the continueUrl parameter is correct with team-gateway

Reply:
👍